test/pyodide_build/test_pywasmcross.py

```diff
@@ -0,0 +1,44 @@
+from collections import namedtuple
+from pathlib import Path
+import sys
+
+sys.path.append(str(Path(__file__).parents[2]))
+
+from pyodide_build.pywasmcross import handle_command  # noqa: E402
+
+
+def _args_wrapper(func):
+    """Convert function to take as input / return a string instead of a
+    list of arguments
```

Comment:
This is mostly for convenience to allow asserting,
```py
handle_command_wrap('gcc -shared -c test.o -o test.so', args)
```
instead of,
```py
handle_command(['gcc', '-shared', '-c', 'test.o', '-o', 'test.so'], args)
```
but I can remove it if you find it not helpful.